openai_chat.gemspec

```
@@ -31,8 +31,15 @@ Gem::Specification.new do |spec|
   spec.executables = spec.files.grep(%r{\Aexe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  # Uncomment to register a new dependency of your gem
-  # spec.add_dependency "example-gem", "~> 1.0"
+  # Register dependencies of the gem
+  spec.add_runtime_dependency "mime-types", "~> 3.5"
```

mime-types 3.5 doesn't appear to work with Ruby 2.0.0, but version `3.0` does. Can we make this `3.0` instead to continue to support Ruby 2?

```suggestion
  spec.add_runtime_dependency "mime-types", "~> 3.0"
```